packages/block-library/src/nextpage/edit.js

```diff
@@ -2,11 +2,10 @@
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
+import { Separator } from '@wordpress/components';
 
 export default function NextPageEdit() {
 	return (
-		<div className="wp-block-nextpage">
-			<span>{ __( 'Page break' ) }</span>
-		</div>
+		<Separator customText={ __( 'Page break' ) } />
```

Comment:
At this moment it might be considered as a breaking change to stop using `wp-block-nextpage` class in here. I'm sure that themes already apply their customizations based on this class.

Reply:
Would it be fine to pass a `className` prop to `Separator` and have it render it that way, or do we want to ensure there's a wrapper `div` around whatever the implementation for separator is?